src/sqlfluff/dialects/dialect_clickhouse.py

```diff
@@ -30,11 +31,21 @@
 
 clickhouse_dialect = ansi_dialect.copy_as("clickhouse")
 clickhouse_dialect.sets("unreserved_keywords").update(UNRESERVED_KEYWORDS)
+
+clickhouse_dialect.add(
```

Comment:
Can we just incorporate `BackQuotedIdentifierSegment` into the existing `clickhouse_dialect.add`?

Reply:
Great, thanks! Yeah it's a bit weird how it compiles.